src/views/general-partner-usual-residential-address-territory-choice.njk

```diff
@@ -1,5 +1,23 @@
 {% extends "layout.njk" %}
 
+{% block backLink %}
+  {% if props.data.generalPartner.data.forename %}
```

Comment:
i think we should be passing the cache into the template so we can check if they selected uk or overseas instead of forename

Reply:
This is @lduranteau 's code that I simply moved, but I think it's correct - it's deciding whether to go back to the General Partner Person or Legal Entity page, hence the check against forename.

(It's the Back link - so user may not have even selected UK or Overseas yet...)